index.d.ts

```diff
@@ -28,6 +28,7 @@ export {ConditionalPick} from './source/conditional-pick';
 export {UnionToIntersection} from './source/union-to-intersection';
 export {Stringified} from './source/stringified';
 export {FixedLengthArray} from './source/fixed-length-array';
+export {OneOrMore, NOrMore} from './source/n-or-more';
```

Comment:
The naming on the second one is a bit unfortunate. I keep reading it as `no-r-more`.

Reply:
Would `AtLeastOne` and `AtLeastN` be better names? Or maybe `MoreThanN`?

Reply:
> Would AtLeastOne and AtLeastN be better names?

Yes

> Or maybe MoreThanN?

This has a different meaning than `NOrMore`. (`>` vs `>=`)

Reply:
How about `NonEmpty`?

It could be `NonEmpty` and `MinimumN`.

Reply:
I like `NonEmpty` and `MinimumN`. See 446e3c5.